### PR TITLE
Plan c homepage

### PR DESCRIPTION
**Pull request by EVogel1999**
Adds plan c, and a notice, to the homepage.

Desktop view:
Screen Shot 2020-03-16 at 9 37 02 AM
Mobile view:
Screen Shot 2020-03-16 at 9 37 35 AM

### CODE REVIEW

File: src/app/cube/home/home.copy.ts [hunk 1]
```diff
@@ -13,5 +13,7 @@ export const COPY = {
     CALLTOACTION: `Contribute to CLARK`,
 
     COLLECTION_BLURB: `Learning Objects are grouped into Collections within CLARK.
-     Each collection is unique, has its own peer-review process, and is supported by its own community.`
+     Each collection is unique, has its own peer-review process, and is supported by its own community.`,
+    PLAN_C_BLURB: `Notice Regarding Plan C: Plan C is a new collection that has a streamlined peer review process.
+     It was created to meet the growing need for cyber content developed for online teaching and learning in response to the COVID-19 pandemic.`
```
Comment (pzalep1) [inline src/app/cube/home/home.copy.ts]:
```suggestion
     It was created to meet the growing need for cyber content developed for online teaching and learning in response to the COVID-19 pandemic.`,
```

File: src/app/cube/home/components/collections/collections.component.html [hunk 2]
```diff
@@ -13,7 +13,10 @@
   </div>
   <div class="collections__grid">
     <div *ngFor="let c of collections">
-      <clark-collection-card [collection]="c"></clark-collection-card>
+      <div class="card-wrapper" [ngClass]="{'center': c.abvName === 'plan c'}">
```
Comment (pzalep1) [inline src/app/cube/home/components/collections/collections.component.html]: I think we need to add some kind of marker to tie it in to the notice at the bottom. Like and asterisk or a badge. The two seem a little disconnected.

Comment (EVogel1999) [reply src/app/cube/home/components/collections/collections.component.html]: With asterisk:
Screen Shot 2020-03-16 at 9 55 22 AM

Comment (EVogel1999) [reply src/app/cube/home/components/collections/collections.component.html]: How would you think the badge would look?

**Pull request merged**